palette/src/yuv/ycbcr.rs

```diff
@@ -0,0 +1,34 @@
+//! A quantized color representation.
+use core::marker::PhantomData;
+
+use crate::yuv::{QuantizationFn, YuvStandard};
+
+/// A quantized YUV color value.
+///
+/// Quantization is the process of encoding the analog signal into digital levels. The thus encoded
+/// signal does not necessarily cover all possible levels and may leave some *headroom* in the low
+/// and high ends of the available code symbols.
+#[derive(Debug, PartialEq, Pixel)]
+#[cfg_attr(feature = "serializing", derive(Serialize, Deserialize))]
+#[palette(
+    palette_internal,
+)]
+#[repr(C)]
+pub struct YCbCr<S: YuvStandard, Q: QuantizationFn> {
+    /// The lumnance signal where `0.0f` is no light and `1.0f` means a maximum displayable amount
+    /// of all colors.
+    pub luminance: Q::Output,
+
+    /// The difference between overall brightness and the blue signal. This is centered around
+    /// `0.0f` but its maximum absolute value depends on the standard.
+    pub blue_diff: Q::Output,
```

Comment:
Would it make sense to just shorten these to `blue` and `red`? Otherwise I feel like "diff" is such a common abbreviation that I could let it pass. Once again, arbitrary, but I'm conflicted.

Reply:
`blue` alone would be confusing. If `diff` is too generic then `blue_chroma` would be an alternative.

Reply:
Nah, leave it as it is. It's better than the alternatives.